validation/standard/standard-immutables.toml

```diff
@@ -8,10 +8,10 @@
 #
 # "immutableReferences":{"85798":[{"start":178,"length":32},{"start":1771,"length":32}]}}
 #
-[releases."op-contracts/v1.8.0-rc.3"] # Holocene https://github.com/ethereum-optimism/optimism/releases/tag/op-contracts%2Fv1.8.0-rc.3
+["op-contracts/v1.8.0-rc.3"] # Holocene https://github.com/ethereum-optimism/optimism/releases/tag/op-contracts%2Fv1.8.0-rc.3
```

Comment:
This was a big footgun. In `standard-versions-*.toml`, the tables are nested under "releases", but in the other TOML files they are not.

Reply:
Any reason we couldn't/shouldn't do the same for `standard-versions-mainnet.toml` and `standard-versions-sepolia.toml`?

Reply:
Done in https://github.com/ethereum-optimism/superchain-registry/pull/768/commits/656b480c77d31e90c5663f42a64cb1d8be5a855c